clients/src/main/resources/common/message/BrokerRegistrationRequest.json

```diff
@@ -53,6 +53,8 @@
     { "name": "Rack", "type": "string", "versions": "0+", "nullableVersions": "0+",
       "about": "The rack which this broker is in." },
     { "name": "IsMigratingZkBroker", "type": "bool", "versions": "1+", "default": "false",
-      "about": "If the required configurations for ZK migration are present, this value is set to true" }
+      "about": "If the required configurations for ZK migration are present, this value is set to true" },
+    { "name": "LogDirs", "type":  "[]uuid", "versions":  "2+",
+      "about": "Log directories configured in this broker which are available." }
```

Comment:
Yeah, it's a good question. I'd say leave it out for now since we're not using it yet. When we do use it, it will have to be integrated with Raft (so that we "know" that voter 2 with ID B is different than voter 2 with ID A, for example).

We can always bump the IBP again when we actually use this on controllers (and when we do actually use this, it might have to go into FetchRequest? Not sure.)